evictionfree/tests/test_schema.py

```diff
@@ -136,12 +148,40 @@ def test_it_works_for_national_users(self):
         assert SCAFFOLDING_SESSION_KEY not in request.session
 
     def test_it_works_for_nyc_users(self, smsoutbox, mailoutbox):
+        request = self.graphql_client.request
+        self.populate_phone_number()
+        update_scaffolding(request, self.NYC_SCAFFOLDING)
+        assert SCAFFOLDING_SESSION_KEY in request.session
+        assert self.execute()["errors"] == []
+        user = JustfixUser.objects.get(phone_number="5551234567")
+        assert user.first_name == "zlorp"
+        assert user.last_name == "zones"
+        assert user.preferred_first_name == ""
+        assert user.email == "zlorp@zones.com"
+        oi = user.onboarding_info
+        assert oi.non_nyc_city == ""
+        assert oi.borough == "MANHATTAN"
+        assert oi.state == "NY"
+        assert oi.address == "123 boop way"
+        assert oi.apt_number == "3B"
+        assert oi.agreed_to_norent_terms is False
+        assert oi.agreed_to_justfix_terms is False
+        assert oi.agreed_to_evictionfree_terms is True
+
+        # This will only get filled out if geocoding is enabled, which it's not.
+        assert oi.zipcode == ""
+
+        assert get_last_queried_phone_number(request) is None
+        assert OnboardingStep1V2Info.get_dict_from_request(request) is None
+        assert SCAFFOLDING_SESSION_KEY not in request.session
+
+    def test_it_works_for_nyc_users_legacy_v2(self, smsoutbox, mailoutbox):
```

Comment:
The goal is to eventually get rid of these legacy items, right? What kind of timeline do you think is reasonable?

Is this related to caches expiring, or is there another relevant milestone?

Reply:
Cool! I think that we should do it soon. Since we store user sessions on the server-side, we can actually just iterate through them to see if any of them match the old legacy schema, which will allow us to remove this code with a lot of confidence that it won't negatively affect users.